coderedcms/templates/coderedcms/pages/article_page.html

```diff
@@ -5,7 +5,7 @@
 {% block content %}
     <article class="codered-article {%if self.cover_image %}has-img{% endif %}">
         {% if self.cover_image %}
-        {% image self.cover_image fill-2000x1000 as cover_image %}
+        {% image self.cover_image original as cover_image %}
```

Comment:
In this case `fill-2000x1000` is going to scale/crop the image to force all blog cover images to be the same size.

I think this is probably the desired default behavior, but our template should make it easier to override. Maybe we should wrap these couple lines of image code in `{% block cover_image %}` or something, so that it can be overridden without needing to override everything?